### ISSUE TITLE
Clarification on R10.4 and polya

### ISSUE THREAD

**Issue by bnpapas** [jts/nanopolish#1144]
It is clear that eventalign-dependent modules does not support R10.4, but it looks like polya uses non-eventaligned durations. Does this mean that polya should be fine to run on R10.4 data, or should it be considered untrustworthy?

**Comment by jts** [comment]
The polyA module is only for direct RNA sequencing, not DNA, so it doesn't work with any of the DNA chemistries (R9.4, or R10.4). Since poly-A calling is now available in dorado, we have not updated the polyA caller for RNA004. I hope that helps.

**Comment by bnpapas** [comment]
Yes, though it is partly my fault for not having a good handle on the nomenclature Nanopore uses - I thought the R was for RNA. Can results still be trusted for RNA004 kits?

**Comment by jts** [comment]
No, you should use dorado for RNA004On Sep 4, 2024, at 1:03 PM, bnpapas ***@***.***> wrote:﻿
Yes, though it is partly my fault for not having a good handle on the nomenclature Nanopore uses - I thought the R was for RNA. Can results still be trusted for RNA004 kits?
